src/github.com/getlantern/flashlight/client/reverseproxy.go

```diff
@@ -18,7 +18,9 @@ import (
 func (client *Client) newReverseProxy(bal *balancer.Balancer) *httputil.ReverseProxy {
 	transport := &http.Transport{
 		TLSHandshakeTimeout: 40 * time.Second,
+		MaxIdleTime:         30 * time.Second,
```

Comment:
It strikes me that we should probably be using TLS session caching here too @oxtoacart no? Not related to this PR of course, but just snooping.